### PR TITLE
Cache Clearing Form - issue 3

### PR DESCRIPTION
**Pull request by cam8001**

### CODE REVIEW

File: src/Form/AkamaiCacheControlForm.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\akamai\Form\AkamaiConfigForm.
```
Comment (nerdstein) [inline src/Form/AkamaiCacheControlForm.php]: Wrong annotation

File: src/Form/AkamaiCacheControlForm.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    );
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    drupal_set_message($this->t('Interact with Akamai API.'));
```
Comment (nerdstein) [inline src/Form/AkamaiCacheControlForm.php:75]: Incomplete? Is this waiting on another ticket?

File: src/Form/AkamaiCacheControlForm.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('akamai.settings');
+    $form = array();
+
+    $form['paths'] = array(
```
Comment (nerdstein) [inline src/Form/AkamaiCacheControlForm.php:34]: Should this be required?

File: src/Form/AkamaiCacheControlForm.php (hunk 2 above)
Comment (cam8001) [inline src/Form/AkamaiCacheControlForm.php:75]: Yep, I think we need to wait until we actually have a service to interact with, ala https://github.com/d8-contrib-modules/akamai/issues/4

I am planning to finish up all the other issues and if @grasmash has implemented the service by then, I will integrate. If not, I will have a go at writing the service myself.

File: src/Form/AkamaiCacheControlForm.php (hunk 3 above)
Comment (cam8001) [inline src/Form/AkamaiCacheControlForm.php:34]: Good call, added.

**Comment by nerdstein** [comment]
Merged

**Pull request merged**